recipes/r-kit/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set version = '0.0.10' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-kit
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/kit_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/kit/kit_{{ version }}.tar.gz
+  sha256: 8a56c1773c9da21e2814e26c28d1c1f64a876f5aa2b1b6e62dbc3a56d2dffa15
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}              # [not win]
+    - {{ compiler('m2w64_c') }}        # [win]
+    - {{ posix }}filesystem        # [win]
+    - {{ posix }}sed               # [win]
+    - {{ posix }}grep              # [win]
+    - {{ posix }}autoconf
+    - {{ posix }}automake          # [not win]
+    - {{ posix }}automake-wrapper  # [win]
+    - {{ posix }}pkg-config
+    - {{ posix }}make
+    - {{ posix }}coreutils         # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
```

Comment:
Those two should be host only as they use the run_exports mechanism to declare run time dependencies:
```suggestion
```